### PR TITLE
Adding initial farm command implementation

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

File: src/cmds/farm.py [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+    return signage_points
+
+
+async def challenges(farmer_rpc_port: int) -> None:
+    signage_points = await get_challenges(farmer_rpc_port)
+    if signage_points is None:
+        return
+
+    signage_points.reverse()
+    for signage_point in signage_points:
```
Comment (jespino) [inline src/cmds/farm.py:180]: I don't know if we want to limit the number of challenges here, I can just add a `[:20]` here for example. Or I can allow to pass a --limit parameter with a default value of `X` and allow to pass 0 for now limit, or something like that. Any suggestion?

File: src/cmds/farm.py [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+from src.util.default_root import DEFAULT_ROOT_PATH
+from src.util.ints import uint64, uint16
+from src.cmds.units import units
+from decimal import Decimal
+from src.wallet.util.transaction_type import TransactionType
+
+MINUTES_PER_BLOCK = (24 * 60) / 4608  # 0.3125
+
+
+def compute_wallets_stats(wallets):
```
Comment (jespino) [inline src/cmds/farm.py:27]: This is based on what I found in the webapp.

File: src/cmds/farm.py (hunk 1 above)
Comment (hoffmang9) [reply src/cmds/farm.py:175]: a default of 20 and the ability to set a higher or lower one is the right way to go.

File: src/cmds/farm.py (hunk 2 above)
Comment (hoffmang9) [reply src/cmds/farm.py:22]: +1

**Review by mariano54 — COMMENTED** [review]
Wow this is awesome @jespino!  Thanks for the hard work.

This change will really come in handy for people farming with cli

**Comment by hoffmang9** [comment]
I'm going to go ahead and merge this. If you want to make that one change, just open an additional PR.

Thanks again!

**Pull request merged**